src/wallet/rpcwallet.cpp

```diff
@@ -813,10 +812,9 @@ static UniValue sendmany(const JSONRPCRequest& request)
                     },
                     {"replaceable", RPCArg::Type::BOOL, /* default */ "wallet default", "Allow this transaction to be replaced by a transaction with higher fees via BIP 125"},
                     {"conf_target", RPCArg::Type::NUM, /* default */ "wallet default", "Confirmation target (in blocks)"},
-                    {"estimate_mode", RPCArg::Type::STR, /* default */ "UNSET", "The fee estimate mode, must be one of:\n"
-            "       \"UNSET\"\n"
-            "       \"ECONOMICAL\"\n"
-            "       \"CONSERVATIVE\""},
+                    {"estimate_mode", RPCArg::Type::STR, /* default */ "UNSET", std::string() + "The fee estimate mode, must be one of:\n"
+            "       \"" + FeeModes("\"\n"
+            "       \"") + "\""},
```

Comment:
You don't need to pass in any whitespace here. Whitespace is done by RPCHelpMan

Reply:
Point. Fixed.